debbuild.spec

```diff
@@ -13,7 +13,7 @@ Name:           debbuild
 Summary:        Build Debian-compatible .deb packages from RPM .spec files
 Version:        24.12.0
 Release:        0%{?dist}
-%if "%{_vendor}" == "debbuild"
+%if 0%{?_debbuild}
```

Comment:
This is not a number.

Reply:
you are right, debbuild doesn't care though

Reply:
Anything that uses a normal rpm spec parser will, though. And this spec file is more or less the one that goes into OBS.